shell/app/modules/project/pages/homepage/readme-markdown.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import i18n from 'core/i18n';
+import { Tooltip } from 'antd';
+import { MarkdownEditor } from 'common';
+import remarkGfm from 'remark-gfm';
+import { useUpdate } from 'common/use-hooks';
+import ReactMarkdown from 'react-markdown';
+import { ScalableImage } from 'common/components/edit-field';
```

Comment:
set `ScalableImage` as a static property of edit-field, then u can write as 

```
import { EditField } from 'common';

const { ScalableImage }  = EditField;
```

Dont import from specific path of common